setup.py

```diff
@@ -4,18 +4,21 @@
 with open('README.md') as f:
     readme = f.read()
 
-with open('LICENSE') as f:
-    license_ = f.read()
-
 setup(
     name='cellmlmanip',
     version='0.1.0',
     description='CellML loading and model equation manipulation',
     long_description=readme,
+    long_description_content_type='text/markdown',
     author='Asif U Tamuri, Sarah M Keating, Maurice Hendrix, Michael Clerx, Jonathan Cooper',
     author_email='j.p.cooper@ucl.ac.uk',
     url='https://github.com/ModellingWebLab/cellmlmanip',
-    license=license_,
+    classifiers=[
+        "Programming Language :: Python :: 3",
```

Comment:
And here

Reply:
yes but are you ok with the change in principle?